### PR TITLE
add some tests

### PR DESCRIPTION
**Pull request by qperrot**
Some more tests

### CODE REVIEW

File: contracts/test/token/ERC677/ERC677.test.ts [hunk 1]
```diff
@@ -122,5 +122,20 @@ describe('ERC677', function () {
         expect(uint256.uint256ToBN(balance1)).to.deep.equal(toBN(0))
       }
     })
+    it('throws an error when sending to 0 address', async () => {
```
Comment (ivpavici) [inline contracts/test/token/ERC677/ERC677.test.ts:126]: newline :)

File: contracts/test/ocr2/proxy.test.ts [hunk 2]
```diff
@@ -28,8 +29,6 @@ describe('proxy.cairo', function () {
     })
 
     console.log(proxy.address)
```
Comment (ivpavici) [inline contracts/test/ocr2/proxy.test.ts:31]: is this console.log necessary still?

Comment (qperrot) [reply contracts/test/ocr2/proxy.test.ts:31]: It was already here

Comment (ivpavici) [reply contracts/test/ocr2/proxy.test.ts:31]: ok, then let's leave these comment for Kristijan to see!

**Review by archseer — APPROVED** [review]

**Pull request merged**